### DISCUSSION TITLE
Commits coming from another account (not mine)

### DISCUSSION THREAD

**Discussion by samjtro** [community/community#24068] [Apps, API and Webhooks]
<p>hello all,</p>
<p>i’ve never seen this error before, but whenever i push a commit to my repos it says it came from from another github user i have no connection with. committing from git on linux, and authenticating through kdewallet which has my github username/personal access token stored. i have reset my PAT and generated a new one; the issue is persisting. im not entirely certain what to do about this or if it is even an issue.</p>
<p>any assistance would be appreciated.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Most likely the author email address (<code>user.config</code>) on your Linux system isn’t configured correctly, and happens to match an address that other user is using. See also: <a href="https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user" class="inline-onebox">Why are my commits linked to the wrong user? - GitHub Docs</a></p>
<p>Check the email address in <code>git log</code> output, and make sure it’s actually</p>
<ol>
<li>one of yours and</li>
<li>registered on your account.</li>
</ol>
